.teamcity/src/main/kotlin/common/Branch.kt

```diff
@@ -0,0 +1,21 @@
+package common
+
+import jetbrains.buildServer.configs.kotlin.v2019_2.DslContext
+
+enum class Branch(val buildTypeId: String) {
+    Master("Master_Check"),
```

Comment:
I don't understand why we need this. I think we need to distinguish between `master` and other branches, since the promotion jobs will be different.

We also don't need  the the build type ids, I think. We create the `Master` project manually on TC, and then add the version settings to it, so the prefix is always a relative `Check` for the Check projects, right?

So I think this shouldn't be an enum. It probably should have a default.

What I don't understand how we select the branch to build from the triggers. Do we need to configure a different VCS somewhere?

Reply:
`buildTypeId` was for backward compatibility, `master`'s buildTypeId was `Gradle_Check` instead of `Gradle_Maste_Check`. Now we keep it separate from old `Gradle_Check`, so this is no longer needed.